docs/plugins/subsonicupdate.rst

```diff
@@ -30,6 +36,14 @@ Configuration
 
 The available options under the ``subsonic:`` section are:
 
+- **url**: The Subsonic server resource. Default: ``http://localhost:4040``
+
+Example: ``https://mydomain.com:443/subsonic``
+
+\* Note: context path is optional
```

Comment:
I think this is OK to delete? We do already give an example that doesn't have an extra path.

Reply:
I agree. Just want to be thorough here. But I imagine beets isn't for the weak hearted anyhow 😛